functions/snippets/pom.xml

```diff
@@ -30,6 +30,11 @@
       <artifactId>functions-framework-api</artifactId>
       <version>1.0.0-alpha-2-rc3</version>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
```

Comment:
I don't think we should need this dependency. Instead of this:

```java
      String requestString = IOUtils.toString(request.getReader());
      JsonElement requestParsed = new JsonParser().parse(requestString);
```

we can just do this:

```java
      JsonElement requestParsed = JsonParser.parseReader(request.getReader());
```